src/components/figures/figure/figure.ts

```diff
@@ -210,9 +209,18 @@ export class FigureComponent extends Component<Props, SpreadsheetChildEnv> {
           y: figure.y + delta[1],
         });
         ev.preventDefault();
-        ev.stopPropagation();
         break;
+      case "Ctrl+A":
+        ev.preventDefault();
+        break;
+      case "Ctrl+Y":
+      case "Ctrl+Z":
+        // Allow the event to bubble to the grid only if Ctrl+Y or Ctrl+Z is pressed
```

Comment:
since https://github.com/odoo/o-spreadsheet/pull/1803 we try an approach where every single component is responsible of their own action and do not rely on their parent potential reaction.

I would rather reimplement the call to REQUEST_UNDO/REDO (or maybe have a function defined in the spreadsheet env ) and call it explicitely

Reply:
@rrahir, I believe I fixed this issue similarly in the initial PR #3678. However, I recall @hokolomopo advised me to allow events to propagate up the grid. Here is the discussion: [Link](https://github.com/odoo/o-spreadsheet/pull/3678#discussion_r1497524202). Perhaps, I will explore the option of reverting back to the previous implementation.

Reply:
I wrote a message on the PR and will try to bring  this discussion up on monday with the whole team present ;-) Your approach is not worse than it was before but I fear we took some liberties that we shouldn't have and now as as good time as any to correct things. Let's keep this PR on hold until we discuss this so you avoid doing the same work twice.

Reply:
We still haven't decided exactly how to handle things but I would like to follow this commit idea : https://github.com/odoo/o-spreadsheet/commit/3660bf27522121e720212f90d15309ba61ae510f#diff-91190cae1bee7c5780c36f202f34ede615a061931ff957bf339ba10eee08f9ee which was to simply handle what you should  leave the other behaviours alone. This means that you should not stop the propagation by default but only if you handle the shortcut.

Reply:
@rrahir, Changes made!